### ISSUE TITLE
Enchantments won't refresh / Enchant takes much more lapis

### ISSUE THREAD

**Issue by I3alaa** [Puyodead1/EnchantCrystals#6]
Throughout some testing it seems I have found 2 bugs.

#1 Enchantments in the Enchanting table are not refreshing.
With settings:
```
Enchanting_tables:
enabled: true
required_lapis: true

merchants: true
```

#2 When Enchanting in the Enchanting Table, depending on the level it can take 1-32 lapis per Enchanting.

Attached a video for reference.
https://youtu.be/tOjLaH6mFv0

**Comment by Puyodead1** [comment]
Thank you for the report, I will look into this in a little bit. 

In the meantime, could you please answer a few quick questions?

What server software are you using and what MC version?

Thanks!

**Comment by Puyodead1** [comment]
Both issues have been fixed in commit f6363433315afe239af869ba007e30ec61997356, new version will be posted to SpigotMC soon
